processor.go

```diff
@@ -331,16 +332,25 @@ func removeOrdered(slice []prompb.Label, s int) []prompb.Label {
 
 func (p *processor) processTimeseries(ts *prompb.TimeSeries) (tenant string, err error) {
 	idx := 0
+
+	configuredLabels := p.cfg.Tenant.LabelList
+
+	if len(configuredLabels) == 0 {
```

Comment:
One more nitpick and we're good to go - can we build this configuredLabels slice prematurely (during config load?) instead of appending to it during each request?

Reply:
Yup good call. This should be addressed in https://github.com/blind-oracle/cortex-tenant/pull/61/commits/04e70f65dbcd1490263199256d84b810e46f303b.